packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/zuora_source
-    version: [">=0.2.0", "<0.3.0"]
\ No newline at end of file
+  # - package: fivetran/zuora_source
+  #   version: [">=0.2.0", "<0.3.0"]
+  - git: https://github.com/fivetran/dbt_zuora_source.git
+    revision: bug/default-cols
+    warn-unpinned: false
```

Comment:
Reminder to update before merge. Actually, I don't think we need to even point at the branch for this update since the transform does not rely on the source changes.

Reply:
Yes, it's not necessary! I just did it so the customer could try out all the changes in one place.